flang/unittests/Optimizer/CMakeLists.txt

```diff
@@ -1,30 +1,13 @@
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 
 set(LIBS
-  FIRAnalysis
-  FIRCodeGen
   FIRDialect
-  FIRTransforms
   ${dialect_libs}
-  MLIRIR
-  MLIRLLVMIR
-  MLIRPass
-  MLIRStandardToLLVM
-  MLIRTransforms
-  MLIRAffineToStandard
-  MLIRAnalysis
-  MLIRSCFToStandard
-  MLIREDSC
-  MLIRParser
-  MLIRStandardToLLVM
-  MLIRSupport
-  MLIRVectorToLLVM
```

Comment:
Where these libraries needed for `Basic.cpp`? Will they be needed for other unit tests?

Reply:
No idea about dependency for other tests.
I figured out later that `${dialect_libs}` does the work.